### PR TITLE
CVAT ubuntu

### PR DESCRIPTION
**Pull request by thomasyu888**
Unfortunately, I was told today that CVAT is only compatible with ubuntu when using GPUs....

* This needs to be behind the load balancer for the address `cvat...synapse.org`. Remove the other instance from this address

### CODE REVIEW

File: config/prod/tom-cvat-ubuntu.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t3.nano" (other available types https://aws.amazon.com/ec2/pricing/on-demand/)
+  InstanceType: "p3.2xlarge"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "500"
+  # (Optional) Base instance on AMI (default: ami-0a313d6098716f372)
+  # AMIId: "ami-0565af6e282977273"
+  # (Optional) Run EC2 in a subnet (default: PrivateSubnet) (valid values: PrivateSubnet or PublicSubnet)
+  VpcSubnet: "PublicSubnet"
```
Comment (zaro0508) [inline config/prod/tom-cvat-ubuntu.yaml]: The load balancer will be the public endpoint so we want the instance to be private.  just comment this out.

Comment (zaro0508) [reply config/prod/tom-cvat-ubuntu.yaml]: Also you can create a PR similar to this to setup a load balancer for your instance. https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/155

Comment (thomasyu888) [reply config/prod/tom-cvat-ubuntu.yaml]: Sorry it isn't clear what I Need to add...

File: config/prod/tom-cvat-ubuntu.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  VolumeSize: "500"
+  # (Optional) Base instance on AMI (default: ami-0a313d6098716f372)
+  # AMIId: "ami-0565af6e282977273"
+  # (Optional) Run EC2 in a subnet (default: PrivateSubnet) (valid values: PrivateSubnet or PublicSubnet)
+  # VpcSubnet: "PublicSubnet"
+  # (Optional) Set true to enable daily backups (default: false)
+  # BackupEc2: "true"
+  # (Optional) Number of daily backups to keep (default: 7) (valid range: 1-180)
+  # SnapshotCount: "14"
+  Ec2InstanceId: "i-03510ca53eb38da83"
```
Comment (thomasyu888) [inline config/prod/tom-cvat-ubuntu.yaml]: Are these the right configurations?

File: config/prod/tom-cvat-ubuntu.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  AppPort: "80"         # rstudio port
+  Subnets: "subnet-21a68d7c,subnet-4f3e2b2b,subnet-77d1e458"
+  # Integration with our jumpcloud directory service (do not change)
+  JcConnectKey: !ssm /infra/JcConnectKey
+  JcServiceApiKey: !ssm /infra/JcServiceApiKey
+  JcSystemsGroupId: !ssm /infra/JcSystemsGroupId
+
+sceptre_user_data:
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  OpenPorts: ["80","8082"]
```
Comment (thomasyu888) [inline config/prod/tom-cvat-ubuntu.yaml]: Will my shiny application still be able to communicate with my CVAT application which is hosted at 8082

Comment (zaro0508) [reply config/prod/tom-cvat-ubuntu.yaml]: shiny and cvat are both docker apps running on the same instance so you can manage the ports between them yourself.

File: config/prod/tom-cvat-ubuntu.yaml (hunk 2 above)
Comment (zaro0508) [reply config/prod/tom-cvat-ubuntu.yaml]: no, you don't want these.  you'll need to setup the load balancer with a separate, follow on PR.

**Pull request merged**